public/backend/js/backend.js

```diff
@@ -1,36 +1,35 @@
-$(document).ready(function(){
-    $('#myTable').DataTable();
+$('div.alert').delay(2500).slideUp();
 
-    $('a.delete').click(function(){
-    	var name = $(this).attr("name");
-    	var url = $(this).attr("url");
-    	swal({   
-    		title: "Are you sure?",   
-    		text: "You want delete this "+ name +"!",   
-    		type: "warning",   
-    		showCancelButton: true,   
-    		confirmButtonColor: "#DD6B55",   
-    		confirmButtonText: "Yes, delete it!",   
-    		closeOnConfirm: false 
-    	}, function(){
+$(document).ready(function () {
+    $('#myTable').DataTable();
 
-    		$.ajaxSetup({
-                 headers: {
-                     'X-CSRF-TOKEN': $('meta[name="_token"]').attr('content')
-                 }
+    $('a.delete').click(function () {
+        var name = $(this).attr("name");
+        var url  = $(this).attr("url");
+        swal({
+            title:"Are you sure?",
```

Comment:
[rank A] we are using language files to define all messages, right? So, please thinking about the way put this to language file too.